src/RequestMatcher.php

```diff
@@ -164,6 +164,38 @@ public static function getSiteVariables($site = NULL) {
     return $vars;
   }
 
+  /**
+   * Gets the variant host.
+   *
+   * @param string $variant
+   *   Variant name.
+   *
+   * @return string|null
+   *   Variant url or null if variant incorrect.
+   */
+  public function getHostForSiteVariant($variant) {
+    if (!in_array($variant, $this->variants)) {
+      return NULL;
+    }
+    $site_variables = static::getSiteVariables();
+    $variant_host = $variant . $this->variantSeparator . $site_variables['SITE_HOST'];
+
+    return $variant_host;
+  }
+
+  /**
+   * Gets the url scheme.
+   *
+   * @return string
+   *   Url scheme value from env var.
+   */
+  public static function getUrlScheme() {
+    if ($url_scheme = getenv('URL_SCHEME')) {
```

Comment:
should be dropped. it's out of scope for this package and URL_SCHEME is not a variable defined by https://github.com/drunomics/multisite-request-matcher